src/System.Runtime/tests/System/TimeSpanTests.cs

```diff
@@ -848,48 +848,135 @@ public static void Subtract_Invalid()
 
         public static IEnumerable<object[]> ToString_MemberData()
         {
+            CultureInfo invariantInfo = CultureInfo.InvariantCulture;
+            CultureInfo commaSeparatorInfo = new CultureInfo("fr-FR");
+            commaSeparatorInfo.NumberFormat.NegativeSign = "!@#!@#"; // validate this is ignored
+
             var input = new TimeSpan(123456789101112);
+            yield return new object[] { input, "%d", invariantInfo, "142" };
+            yield return new object[] { input, "dd", invariantInfo, "142" };
+            yield return new object[] { input, "%h", invariantInfo, "21" };
+            yield return new object[] { input, "hh", invariantInfo, "21" };
+            yield return new object[] { input, "%m", invariantInfo, "21" };
+            yield return new object[] { input, "mm", invariantInfo, "21" };
+            yield return new object[] { input, "%s", invariantInfo, "18" };
+            yield return new object[] { input, "ss", invariantInfo, "18" };
+            yield return new object[] { input, "%f", invariantInfo, "9" };
+            yield return new object[] { input, "ff", invariantInfo, "91" };
+            yield return new object[] { input, "fff", invariantInfo, "910" };
+            yield return new object[] { input, "ffff", invariantInfo, "9101" };
+            yield return new object[] { input, "fffff", invariantInfo, "91011" };
+            yield return new object[] { input, "ffffff", invariantInfo, "910111" };
+            yield return new object[] { input, "fffffff", invariantInfo, "9101112" };
+            yield return new object[] { input, "%F", invariantInfo, "9" };
+            yield return new object[] { input, "FF", invariantInfo, "91" };
+            yield return new object[] { input, "FFF", invariantInfo, "91" };
+            yield return new object[] { input, "FFFF", invariantInfo, "9101" };
+            yield return new object[] { input, "FFFFF", invariantInfo, "91011" };
+            yield return new object[] { input, "FFFFFF", invariantInfo, "910111" };
+            yield return new object[] { input, "FFFFFFF", invariantInfo, "9101112" };
+            yield return new object[] { input, "dd\\.ss", invariantInfo, "142.18" };
+            yield return new object[] { input, "dd\\.ss", commaSeparatorInfo, "142.18" };
```

Comment:
Maybe more `d`'s so as to get leading zeros?

Reply:
Added.